apis/v1alpha1/cluster_tc_program_types.go

```diff
@@ -33,11 +33,11 @@ type ClTcAttachInfo struct {
 	// interfaceSelector to determine the network interface (or interfaces)
 	InterfaceSelector InterfaceSelector `json:"interfaceSelector"`
 
-	// containers identifies the set of containers in which to attach the eBPF
-	// program. If Containers is not specified, the BPF program will be attached
-	// in the root network namespace.
+	// networkNamespaces identifies the set of network namespaces in which to
+	// attach the eBPF program. If networkNamespaces is not specified, the BPF
+	// program will be attached in the root network namespace.
 	// +optional
-	Containers *ClContainerSelector `json:"containers"`
+	NetworkNamespaces *ClNetworkNamespaceSelector `json:"networkNamespaces"`
```

Comment:
do u need to add `omitempty` here

Reply:
I tried to add omitempty everywhere it should be.